shared/version/api.go

```diff
@@ -383,6 +383,7 @@ var APIExtensions = []string{
 	"operations_get_query_all_projects",
 	"metadata_configuration",
 	"syslog_socket",
+	"lifecycle_event_name_and_project",
```

Comment:
mismatch of api extension names between the 2 files here, lets go with the the one in doc/api-extensions.md